src/lib/data/alignment.js

```diff
@@ -698,26 +715,40 @@ export default class Alignment {
       return {
         segment,
         tokenIndex,
-        mergePosition: 'left',
+        position: TokensEditController.direction.LEFT,
         tokenMergeTo: segment.getTokenByIndex(tokenIndex + 1)
       }
     }
     return false
   }
 
   splitToken (token, tokenWord) {
-    if (this.tokenIsGrouped(token)) { return false }
-
     const segment = this.getSegmentByToken(token)
     const tokenIndex = segment.getTokenIndex(token)
     const alignedText = this.getAlignedTextByToken(token)
 
-    const newIdWord = alignedText.getNewIdWord(segment)
+    const newIdWord1 = alignedText.getNewIdWord({

```

Comment:
I don't think this implements the requirements exactly as I was thinking in https://github.com/alpheios-project/alignment-editor-new/issues/39#issuecomment-758387065 . I just realized however, that you might not have seen them fully, because I forgot to escape angle brackets in the markdown.  Can you take another look?

I was thinking that each change would need an automatic increment, and an aggregating format so that we could track subsequent changes to the same token. I think maybe we need some test cases to see the impact of a whole sequence of changes to get clear on how this should work though.

Reply:
Would update according to the explanation

Reply:
Done